katas/content/superposition/greenberger_horne_zeilinger/solution.md

```diff
@@ -0,0 +1,23 @@
+### Solution
+
+The single-qubit GHZ state is the plus state $\frac{1}{\sqrt{2}} \big (|0\rangle + |1\rangle\big)$ that we've discussed in the first task. As a reminder, that state is prepared by applying a Hadamard gate.
+
+The 2-qubit GHZ state is the Bell state $\frac{1}{\sqrt{2}} \big (|00\rangle + |11\rangle\big)$ that we've discussed in the two previous tasks.
+
+The next one is the 3-qubit GHZ state:
+$$|GHZ\rangle = \frac{1}{\sqrt{2}} \big (|000\rangle + |111\rangle\big)$$
+
+Let's use the 2-qubit state as a building block to construct the state for 3 qubits. First, let's add a third qubit to the above state (on the right from the first two qubits).
+Comparing this state with the desired end state, we see that they differ only in the third (rightmost) qubit:
+
+$$|\Phi^+\rangle |0\rangle = \frac{1}{\sqrt{2}} \big (|000\rangle + |11\color{red}0\rangle\big)$$
+$$|GHZ\rangle = \frac{1}{\sqrt{2}} \big (|000\rangle + |11\color{red}1\rangle\big)$$
```

Comment:
I didn't leave a comment on this in the first round of review, sorry! I don't think `\color` tag works well in the new interface, it colors the whole end of the formula instead of just the piece that is needed, so we probably want to drop it. Making it bold instead works well
```suggestion
$$|\Phi^+\rangle |0\rangle = \frac{1}{\sqrt{2}} \big (|000\rangle + |11\textbf{0}\rangle\big)$$
$$|GHZ\rangle = \frac{1}{\sqrt{2}} \big (|000\rangle + |11\textbf{1}\rangle\big)$$
```